### DISCUSSION TITLE
I get a notification that my student ID card is already loaded and I can't sign in as a student

### DISCUSSION THREAD

**Discussion by Senaekmen** [community/community#113668] [GitHub Education]
### Select Topic Area

Question

### Body

" Your uploaded proof has already been used previously " I get an error saying the evidence you uploaded has been used before.

**Comment by brkykb** [comment]
Bro wait a little bit they will check your ıd after that they can apply you.

**Comment by krishnasunny** [comment]
Bro that message usually pops up if you're trying to upload something you've already sent in before. If you're sure you need to use the same document again, maybe there's a way to let them know or add a note explaining why. If that doesn't work, reaching out to their support might help clear things up!

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Senaekmen , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @Senaekmen 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
